kata-deploy/scripts/kata-deploy.sh

```diff
@@ -12,6 +12,8 @@ crio_conf_file="/etc/crio/crio.conf"
 crio_conf_file_backup="${crio_conf_file}.bak"
 containerd_conf_file="/etc/containerd/config.toml"
 containerd_conf_file_backup="${containerd_conf_file}.bak"
+containerd_k3s_conf_file="/var/lib/rancher/k3s/agent/etc/containerd/config.toml"
```

Comment:
@wilsonianb what if we are just changing the yaml to mount /varl/ib/rancher/k3s/agent/etc/containerd to /etc/containerd, and kept majority of the logic identical? 

AFAIU we'd still need to handle the different service name resets, which unfortunately would be k3s and k3s-agent, depending on whether this was master  or worker node.

Reply:
```
The DaemonSet "kata-deploy" is invalid: spec.template.spec.containers[0].volumeMounts[2].mountPath: Invalid value: "/etc/containerd/": must be unique
```
I might be possible to mount individual files (including backups) as a workaround, but that seems messier than how the volumes are currently mounted

Reply:
Well, I was originally thinking there'd be a kata-deploy-in-k3s.yaml which is just about the same, only utilizing the k3s path rather than the standard /etc/containerd/ path.

Reply:
Latest changes are set up to use kustomize as an alternative to maintaining multiple kata-deploy*.yaml files.